### DISCUSSION TITLE
Logger Output [v5]

### DISCUSSION THREAD

**Discussion by joeneldeasis** [adonisjs/core#4158] [Help]
I'm currently using adonis v5 and already read the logger [guide](https://docs.adonisjs.com/guides/logger) but upon checking the output of the code in the docs below is different from the actual logs output.

Sample code from the docs and the expected output:
```
import Logger from '@ioc:Adonis/Core/Logger'

Logger.info({ username: 'virk' }, 'user signup')
// output: {"username":"virk","msg":"user signup"}
```

What I get:
```
[09:52:35.551] INFO (test-api/4076): user signup
    username: "virk"
```

Here's my `package.json` for your reference.
```
{
  "name": "test-api",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "node ace serve --watch",
    "build": "node ace build --production",
    "start": "node server.js"
  },
  "devDependencies": {
    "@adonisjs/assembler": "^5.9.5",
    "@japa/preset-adonis": "^1.2.0",
    "@japa/runner": "^2.2.2",
    "@types/proxy-addr": "^2.0.0",
    "@types/source-map-support": "^0.5.6",
    "adonis-preset-ts": "^2.1.0",
    "pino-pretty": "^9.1.1",
    "typescript": "~4.6",
    "youch": "^3.2.2",
    "youch-terminal": "^2.1.5"
  },
  "dependencies": {
    "@adonisjs/core": "^5.9.0",
    "@adonisjs/lucid": "^18.3.0",
    "@adonisjs/repl": "^3.1.11",
    "luxon": "^3.2.1",
    "proxy-addr": "^2.0.7",
    "reflect-metadata": "^0.1.13",
    "source-map-support": "^0.5.21"
  }
}
```

**Comment by Julien-R44** [comment]
That's normal, the log that you see displayed is formatted in the way stipulated in the doc ; however when you are in dev mode ( NODE_ENV === "developpement" ), there is pino-pretty which comes to display the logs in a slightly more appealing way : https://github.com/pinojs/pino-pretty

Also see : https://docs.adonisjs.com/guides/logger#logging-in-development

You can disable pino-pretty via this property in the logger configuration, if you want to check the format of your logs :
https://docs.adonisjs.com/guides/logger#prettyprint
